src/components/Table/head/__test__/head.spec.js

```diff
@@ -22,6 +22,16 @@ describe('<Head />', () => {
         const header = component.find('Header');
         expect(header.length).toBe(3);
     });
+    it('should not show the header columns', () => {
+        const component = mount(<Head columns={columns} hideTableHeader />);
+        const header = component.find('Header');
+        expect(header.at(0).prop('hideTableHeader')).toBeTruthy();
+    });
+    it('should show the header columns', () => {
+        const component = mount(<Head columns={columns} />);
+        const header = component.find('Header');
+        expect(header.at(0).prop('hideTableHeader')).toBeFalsy();
+    });
```

Comment:
these tests ^^ should check the header is not visible, checking the prop value when is being passed is testing React :) .. not the component behavior.

Although, the implementation of the feature looks good to me